knowledge-base/grid-customize-dragged-dropped-row.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Customize the dragged and dropped row
+description: How to customize and change the style of the dragged and dropped row?
+type: how-to
+page_title: Customize the dragged and dropped row
+slug: grid-kb-customize-dragged-dropped-row
+position: 
+tags: grid, drag, drop, customize, style
+ticketid: 1547069
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am using Grid with Drag and Drop functionality to drag the items from one grid to another. I want to change the background color of the dragged item. How to achieve that?
+
+## Solution
+
+Do the following to customize the appearance of the dropped item:
+
+* Use a global variable to store the dropped item. You can get it from the `GridRowDropEventArgs` of the [`OnRowDrop` event]({%slug grid-events%}#onrowdrop)
+
+* Handle the [`OnRowRender` event]({%slug grid-events%}#onrowrender) of the Grid to set a custom CSS class to the item that is matching the dropped one (you can compare them by Id, for example).
+
+* Additionally, you can set some custom styles for the drag clue container
+
+The example below demonstrates the described approach. Note that this way, the customized item will only be the one matching the dropped item. Once you drag and drop another item, it will be the one having the custom styles and the previous one will be targeted as regular Grid item.
```

Comment:
```suggestion
Here is an example. Note that the customized item will match only the last dropped item. Once you drag and drop another item, it will assume the custom styles and the previous one will be treated as a regular Grid item. It is possible to enhance the code logic, preserve all dragged rows in a collection and persist the custom styles for all of them.
```